OWNERS

```diff
@@ -1,5 +1,4 @@
 approvers:
```

Comment:
Seriously? Do you guys want to take the entire repository over?

Reply:
My bad, I just took some inspiration from other implementations, I didn't think it was so critical. Could you explain to me the meaning of this? Maybe even tell me what you suggest so I can change it

Reply:
@smongiar each folder has OWNERS file which has members that can approve the changes for that folder, we only add OWNERS file in our added folders, for example, this [MTR PR](https://github.com/openshift/release/pull/36181/files) only added OWNERS in folders created for MTR.
Please revert changes in this OWNERS file.

Reply:
Ok, changes reverted